dask/array/core.py

```diff
@@ -2570,7 +2570,7 @@ def from_array(
 ):
     """ Create dask array from something that looks like an array
 
-    Input must have a ``.shape`` and support numpy-style slicing.
+    Input must have a ``.shape``, ``.ndim`` and support numpy-style slicing.
```

Comment:
In light of https://github.com/dask/dask/issues/5106#issuecomment-512235869, we should probably add the need for a `.dtype` attribute too 

```suggestion
    Input must have a ``.shape``, ``.ndim``, ``.dtype`` and support numpy-style slicing.
```

Reply:
Good idea, thanks @jrbourbeau!